Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:alpine AS build-env
+
+# Install any compile-time system dependencies.
+RUN apk update && \
```

Comment:
Sorry I missed this on my first read though. Can we consolidate this to `apk --no-cache add git curl`

Reply:
Nice catch! I should've looked at one of our other Dockerfiles to remind myself this flag exists.

Reply:
With the `no-cache` flag you don't need the apk update anymore. It handles doing the update for you.